designdocs/privacy-design.md

```diff
@@ -0,0 +1,289 @@
+## Note
+This document was used as background for the [Privacy and Security Explainer](https://github.com/immersive-web/webxr/blob/master/privacy-security-explainer.md) and has been superseded by that explainer.
```

Comment:
Better to make the link relative so that it's not impacted by branch changes.

Reply:
Oh good call, done. Thanks @NellWaliczek